### PR TITLE
WIP: lock down analyzer version

### PR DESCRIPTION
**Pull request by jeffbcross**
This is still causing different issues locally and will probably break Travis, but worth a shot.

### CODE REVIEW

**Comment by jessegood** [comment]
> This is still causing different issues locally

I posted on [gitter](https://gitter.im/angular/angular?at=55e0f210017739836ed4656b) an issue, an the error seemed to be referring to the analyzer. Any chance my problem is related?

**Comment by jeffbcross** [comment]
@jessegood yeah same issue

**Comment by vicb** [comment]
Seems like an issue with the analyzer package: https://github.com/dart-lang/sdk/commit/335b87c701d4176a669a318d50f345d4b7977267#commitcomment-12960070

File: modules/angular2/pubspec.yaml [hunk 1]
```diff
@@ -9,7 +9,7 @@ homepage: <%= packageJson.homepage %>
 environment:
   sdk: '>=1.10.0 <2.0.0'
 dependencies:
-  analyzer: '>=0.24.4 <0.27.0'
+  analyzer: '>=0.25.1 <0.26.1+2'
```
Comment (vicb) [inline modules/angular2/pubspec.yaml]: `<=0.26.1` would probably solve the issue until `0.26.1+3` gets release.

Not sure if we cache Dart dependencies on Travis ? If so, we might also need to clear the cache there.

**Pull request closed without merging**